test/byContract/IexecMaintenance/IexecMaintenance.ts

```diff
@@ -0,0 +1,186 @@
+// SPDX-FileCopyrightText: 2020-2024 IEXEC BLOCKCHAIN TECH <contact@iex.ec>
+// SPDX-License-Identifier: Apache-2.0
+
+import { loadFixture } from '@nomicfoundation/hardhat-network-helpers';
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/signers';
+import { ethers, expect } from 'hardhat';
+import { loadHardhatFixtureDeployment } from '../../../scripts/hardhat-fixture-deployer';
+import {
+    IexecInterfaceNative,
+    IexecInterfaceNative__factory,
+    IexecLibOrders_v5,
+    IexecMaintenanceExtra,
+    IexecMaintenanceExtra__factory,
+} from '../../../typechain';
+import { getIexecAccounts } from '../../../utils/poco-tools';
+
+const randomAddress = () => ethers.Wallet.createRandom().address;
+const configureArgs = [
+    randomAddress(),
+    'some name',
+    'some symbol',
+    100,
+    randomAddress(),
+    randomAddress(),
+    randomAddress(),
+    randomAddress(),
+] as [string, string, string, number, string, string, string, string];
+const hashZero = ethers.constants.HashZero;
+const someDomainSeparator = '0x0000000000000000000000000000000000000000000000000000000000000001';
+
+describe('Maintenance', async () => {
+    let proxyAddress: string;
+    let [iexecPoco, iexecPocoAsAdmin]: IexecInterfaceNative[] = [];
+    let [iexecAdmin, anyone]: SignerWithAddress[] = [];
+
+    beforeEach('Deploy', async () => {
+        proxyAddress = await loadHardhatFixtureDeployment();
+        await loadFixture(initFixture);
+    });
+
+    async function initFixture() {
+        const accounts = await getIexecAccounts();
+        ({ iexecAdmin, anyone } = accounts);
+        iexecPoco = IexecInterfaceNative__factory.connect(proxyAddress, anyone);
+        iexecPocoAsAdmin = iexecPoco.connect(iexecAdmin);
+    }
+
+    describe('Configure', () => {
+        it('Should configure', async () => {
+            await setDomainSeparator(hashZero);
+            await iexecPocoAsAdmin.configure(...configureArgs).then((tx) => tx.wait());
+            expect(await iexecPoco.eip712domain_separator()).equal(
+                await hashDomain(await iexecPoco.domain()),
+            );
```

Comment:
Also compare with hash of local values.

Reply:
Domain is already tested in `Should get domain`